splice_si_api/src/main/java/com/splicemachine/si/impl/server/SITransactor.java

```diff
@@ -184,7 +184,7 @@ private MutationStatus getCorrectStatus(MutationStatus status,MutationStatus old
             constraintState = new SimpleTxnFilter(null, txn, NoOpReadResolver.INSTANCE, txnSupplier);
             supplier = constraintState.getTxnSupplier();
         } else {
-            supplier = new ActiveTxnCacheSupplier(txnSupplier, 100);
+            supplier = new ActiveTxnCacheSupplier(txnSupplier, 100, 4000);
```

Comment:
Why 4000?

Reply:
It was inline with the defaults of 128/4096 , now I get them from the Config if it's available or set it to the defaults if not.